### PR TITLE
Added price breakdown and estimated price to flight booking.

### PR DESCRIPTION
**Pull request by randunel**

### CODE REVIEW

File: src/schema/flight-booking.json [hunk 1]
```diff
@@ -362,6 +370,28 @@
...
+            "type": "array",
+            "description": "The list of price components, as found on the website.",
+            "minItems": 1,
+            "items": { "$ref": "#/domains/FlightBooking/types/PriceBreakdownItem" }
+        },
+        "PriceBreakdownItem": {
+            "type": "object",
+            "description": "A component of the price breakdown.",
+            "properties": {
+                "text": {
```
Comment (inca) [inline src/schema/flight-booking.json]: How about `description`?

Comment (randunel) [reply src/schema/flight-booking.json]: There is `description` on line `386` :/

Comment (inca) [reply src/schema/flight-booking.json]: yup, I meant a property name, but it's not really important, feel free to ignore me 😄

Comment (randunel) [reply src/schema/flight-booking.json]: Oh, you meant `description` in place of `text` :/ IDK if `description` is the right name for this property, as some price components have a q-mark tooltip `?` to show the actual detailed description of the price component :D

**Pull request merged**